src/secur3dit/Main.java

```diff
@@ -0,0 +1,35 @@
+package secur3dit;
+
+import java.awt.GraphicsDevice;
+import java.awt.GraphicsEnvironment;
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.stage.Stage;
+
+/**
+ *
+ * @author naman
+ */
+public class Main extends Application{
+    public static Stage stage = null;
+    public static void main(String[] args) {
+        // TODO code application logic here
```

Comment:
we could get rid of this comment or does this signify that something is pending? @namannihal